status/reporter_gitlab.go

```diff
@@ -158,9 +158,9 @@ func (r *GitLabReporter) setCommitStatus(report TestReport) error {
 			return fmt.Errorf("error while getting all commitStatuses for sha %s: %w", r.sha, err)
 		}
 		existingCommitStatus := r.GetExistingCommitStatus(allCommitStatuses, report.FullName)
-		if existingCommitStatus != nil && existingCommitStatus.Status == string(gitlab.Running) {
-			r.logger.Info("Will not update the existing commit status from `running` to `running`",
-				"scenario.name", report.ScenarioName, "commitStatus.ID", existingCommitStatus.ID)
+		if existingCommitStatus != nil && (existingCommitStatus.Status == string(gitlab.Running) || existingCommitStatus.Status == "enqueue") {
```

Comment:
Do note that this part of the code is only reached if glState == gitlab.Running ([here currently](https://github.com/konflux-ci/integration-service/pull/842/files#diff-cbf7e6d58cc12a67b910b5b9108ba68d206e75cc8560ca0ab2b19e2d34415702R858)).

Reply:
my bad, 
you are right , went to check outside of `glState == "running "` 
if the required update to be `enqueue` then skip the update